docs/en_US/TrainingService/AMLMode.rst

```diff
@@ -82,6 +82,11 @@ Compared with `LocalMode <LocalMode.rst>`__ trial configuration in aml mode have
 
   * required key. The docker image name used in job. NNI support image ``msranni/nni`` for running aml jobs.
 
+Monitor your code in the cloud by using the studio
+--------------------------------------------------
+
+To monitor your job's code, you need visit your studio which you create at step 5. Once the job completes, go to the Outputs + logs tab. There you can see a 70_driver_log.txt file, This file contains the standard output from a run and can be useful when you're debugging remote runs in the cloud. Learn more about aml from `here <https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-1st-experiment-hello-world>`__.
+
```

Comment:
please place this part at the end of this doc, the following config reference also belongs to the section `Run an experiment`.

you need visit -> you need to visit

Reply:
Done.